msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/basic/TestCase2517374.java

```diff
@@ -0,0 +1,101 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.msalonly.basic;
+
+import android.util.Log;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest;
+import com.microsoft.identity.client.ui.automation.annotations.RetryOnFailure;
+import com.microsoft.identity.client.ui.automation.annotations.RunOnAPI29Minus;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.app.MsalTestApp;
+import com.microsoft.identity.client.ui.automation.broker.BrokerCompanyPortal;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandlerParameters;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.List;
+
+
+@RetryOnFailure(retryCount = 2)
+@RunOnAPI29Minus
+@SupportedBrokers(brokers = {BrokerCompanyPortal.class})
```

Comment:
Also, please add LTWTests annotation that was introduced in this PR
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2067